### DISCUSSION TITLE
🐛 ld: bitcode bundle could not be generated

### DISCUSSION THREAD

**Discussion by ghost** [swiftyfinch/Rugby#98] [Q&A] [answered]
## Did you already try Rugby doctor?
Yes

## Describe the bug
1. Rugby cache successful
2. Xcode archive app hit the error
`
ld: bitcode bundle could not be generated because '.rugby/build/Debug-iphoneos/SnapKit/SnapKit.framework/SnapKit' was built without full bitcode. All frameworks and dylibs for bitcode must be generated from Xcode Archive or Install build file '.rugby/build/Debug-iphoneos/SnapKit/SnapKit.framework/SnapKit' for architecture arm64
`

## To Reproduce
```
#Pofile
target 'App' do 
    pod 'SnapKit', '~> 5.0.1'
end
target 'Framework' do 
    pod 'SnapKit', '~> 5.0.1'
end
```
set Build Settings key ENABLE_BITCODE as NO in target App.
set Build Settings key ENABLE_BITCODE as YES in target Framework.
set target Framework into Dependencies and LinkBinary With Libraries in Build Phase in target App 
xcode archive

## Expected behavior
How to control .rugby cache W/ or W/O ENABLE_BITCODE

## Screenshots
N/A

## Environment (please complete the following information):
 - Version [e.g. 1.13.0]
 - Logs (use `rugby doctor`)
 - Podfile or Example project (if you can share these)

## Additional context
N/A

**Comment by swiftyfinch** [comment]
Could you describe this issue with the standard bug template?
I need to know what steps you did to reproduce this issue.

**Comment by ghost** [comment]
> Could you describe this issue with the standard bug template? I need to know what steps you did to reproduce this issue.

updated.

**Comment by swiftyfinch** [comment]
@Wei-Pinkoi Thank you for the description!
There was a discussion about this https://github.com/swiftyfinch/Rugby/discussions/8#discussioncomment-729481

But now I'm not sure, why you build for `Debug-iphoneos`?
Did you try to build like that `rugby cache --config Release` and enable bitcode for this configuration?
Anyway, I don't see there a 🏈 Rugby bug. You can open a new discussion or a feature request.

**Comment by ghost** [comment]
I think that is not the issue related configurations.

I think it's 2 targets use diff ENABLE_BITCODE value, due to the issue solved after values are the same(false) or without Rugby cached.

So I consider that the pre-build pod's framework should support this scenario?

But that's ok, we can close the non-issue.

**Comment by swiftyfinch** [comment]
Yeah, please open a discussion for that reason if you want to.

If you have two different targets with different values I think you just need to change it 🙂
🏈 Rugby does not change how each target should be built.

**Reply by swiftyfinch** [reply]
Oh, I've converted it to the discussion.

**Comment by swiftyfinch** [comment, marked as answer]
@Wei-Pinkoi I've checked the issue and you are right.
I will make a fix soon.

**Reply by swiftyfinch** [reply]
#102
